jsonrpc/debug_endpoint_test.go

```diff
@@ -1070,6 +1075,97 @@ func TestGetRawTransaction(t *testing.T) {
 	}
 }
 
+func TestGetRawReceipts(t *testing.T) {
+	t.Parallel()
+
+	const (
+		cumulativeGasUsed = 28000
+		gasUsed           = 26000
+	)
+
+	rec := createTestReceipt(nil, cumulativeGasUsed, gasUsed, testTx1.Hash())
+	receipts := []*types.Receipt{rec}
+
+	tests := []struct {
+		name   string
+		filter BlockNumberOrHash
+		store  *debugEndpointMockStore
+		result interface{}
+		err    bool
+	}{
+		{
+			name:   "HeaderNotFound",
+			filter: EarliestBlockNumberOrHash,
+			store: &debugEndpointMockStore{
+				getHeaderByNumberFn: func(num uint64) (*types.Header, bool) {
+					assert.Equal(t, uint64(0), num)
+
+					return nil, false
+				},
+			},
+			result: nil,
+			err:    true,
+		},
+		{
+			name:   "ReceiptsNotFound",
+			filter: LatestBlockNumberOrHash,
+
+			store: &debugEndpointMockStore{
+				headerFn: func() *types.Header {
+					return testLatestBlock.Header
+				},
+				getBlockByHashFn: func(hash types.Hash, full bool) (*types.Block, bool) {
+					return testLatestBlock, true
+				},
+				getReceiptsByHashFn: func(hash types.Hash) ([]*types.Receipt, error) {
+					return nil, ErrExecutionTimeout
+				},
+			},
+			result: nil,
+			err:    true,
+		},
+		{
+			name:   "Success",
+			filter: LatestBlockNumberOrHash,
+			store: &debugEndpointMockStore{
+				headerFn: func() *types.Header {
+					return testLatestBlock.Header
+				},
+
+				getBlockByHashFn: func(hash types.Hash, full bool) (*types.Block, bool) {
+					return testLatestBlock, true
+				},
+
+				getReceiptsByHashFn: func(hash types.Hash) ([]*types.Receipt, error) {
+					return receipts, nil
+				},
+			},
+			result: [][]byte{rec.MarshalRLP()},
+			err:    false,
+		},
+	}
+
+	for _, test := range tests {
+		test := test
+
+		t.Run(test.name, func(t *testing.T) {
+			t.Parallel()
+
+			endpoint := NewDebug(test.store, 100000)
+
+			res, err := endpoint.GetRawReceipts(test.filter)
+
+			assert.Equal(t, test.result, res)
```

Comment:
Try and see whether linter will complain, but not necessary to add empty line after each line IMO:

```suggestion
			endpoint := NewDebug(test.store, 100000)
			res, err := endpoint.GetRawReceipts(test.filter)
			assert.Equal(t, test.result, res)
```